src/eve-common/EVE_Defines.h

```diff
@@ -396,7 +396,7 @@
 (itemID == npcTraderJoe)
 
 #define IsTrader(itemID) \
-(((itemID >= minTrader) && (itemID <= maxTrader))
+((itemID >= minTrader) && (itemID <= maxTrader))
```

Comment:
This macro contained an extra parenthetical at the beginning, causing usage of `IsTrader` to fail immediately anywhere. I have fixed it here.